docs/en/intro-to-loom.md

```diff
@@ -185,3 +185,126 @@ yarn serve
+
+Click "Sign" and, under the hood, the dashboard will map your accounts. Next, you will be redirected to your dashboard. It'll look similar to this:
+
+![Map Accounts](/developers/img/deploy-to-loom-dashboard.png)
+
+
+### Deposit Loom to Plasmachain
+
+From the left pane, click on the "Deploy to Plasmachain". You will be redirected to a page similar to the one below:
+
+![New Account Options](/developers/img/deploy-to-loom-deposit-loom.png)
+
+Notice that, in order to deploy to Plasmachain, you need to stake 10 LOOM. Click on the "Deposit more Loom to Plasmachain" and follow the instructions.
+
+### Adding the Public Keys
+
+From here you have two options. You either create a new account, or you use an existing address:
+
+Let's explore both a bit further.
+
+#### Option 1 - Creating a New Account
+
+Say you clicked "Generate New Key". Next, a new window will pop showing the details of the new account:
+
+![New Account Details](/developers/img/deploy-to-loom-create-new-account.png)
+
+We can't emphasize this more: **store the mnemonic somewhere safe**.
+
+Next, copy the public address and paste it into the "Your Loom Public Address" textbox:
+
+![Public Address](/developers/img/deploy-to-loom-paste-public-address.png)
+
+Lastly, click "Add Key" and you're set.
+
+#### Option 2 - Use an Existing Address
+
+Now, as an example, let's suppose you already have a Loom address- `0xC8613234a93977079d464e9b77bF60b227DB67A1`. If so, all you have to do is to replace the `0x` prefix with `loom`.  It'll look something like this: `loomC8613234a93977079d464e9b77bF60b227DB67A1`.
+
+Lastly, click "Add Key" and you're ready to deploy to Loom.
+
+#### Deploying to Loom
+
+If in the previous step, you chose to use an existing address, chances are that you already know how to use your private key to deploy to Loom. Thus, we won't describe that scenario.
+
+However, if you chose to generate a new address, here's what you have to do:
+
+ - Fire up your favorite text editor and open `truffle-config.js`
+
+ - Start by importing a few things:
+
+```js
+import { sha256 } from "js-sha256"
+import { CryptoUtils } from "loom-js"
+import { LoomTruffleProvider } from 'loom-truffle-provider'
+```
+
+ - Next, we'll have to compute the private key from mnemonic as follows:
+
+ ```js
+const privateKeyUint8ArrayFromSeed = await CryptoUtils.generatePrivateKeyFromSeed(new Uint8Array(sha256.array(mnemonic)))
+const privateKeyB64 = await CryptoUtils.Uint8ArrayToB64(privateKeyUint8ArrayFromSeed)
+ ```
+
+- Lastly, we can instantiate `loomProvider` like this:
+
+```js
+new LoomTruffleProvider(chainId, writeUrl, readUrl, privateKeyB64)
+```
+
+Wrapping it up, your `truffle-config.js` file should look something like this:
+
+```js
+
+import { sha256 } from 'js-sha256'
+import { CryptoUtils } from 'loom-js'
+import { LoomTruffleProvider } from 'loom-truffle-provider'
+
+const mnemonic = 'YOUR MNEMONIC'
```

Comment:
This doesn't look right, the mnemonic words need to be converted to a 64-byte seed, then you SHA256 the seed to get the 32 bytes to pass into `generatePrivateKeyFromSeed`

Reply:
`sha256.array(mnemonic)`  will returns 32 bytes

Reply:
@enlight @mimiiiz can't we just use:
```
loomTruffleProvider.createExtraAccountsFromMnemonic(mnemonic, 10);
```

Reply:
Yes, that `createExtraAccountsFromMnemonic` seems reasonable, @mimiiiz can you make the dashboard generate the key like LoomTruffleProvider? https://github.com/loomnetwork/loom-truffle-provider/blob/c5a56aa2e4fda2317be6ca84e736e0f8e28070ff/src/index.ts#L45